### ISSUE TITLE
restart memcached server after upgrade needed

### ISSUE THREAD

**Issue by Netmees** [nextcloud/server#49844]
### How to use GitHub

* Please use the 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to show that you are interested into the same feature.
* Please don't comment if you have no relevant information to add. It's just extra noise for everyone subscribed to this issue.
* Subscribe to receive notifications on status change and new comments. 

### Problem

maibe its posible to add lines to restart memcached or clear cache after upgrade becouse it displays an error if memcached or other cache its used
if its not posible just add a display note about the need of restarting cache system

**Comment by solracsf** [comment]
Why do you believe this is needed? Please detail.

**Comment by Netmees** [comment]
just to dont make people loosing time after uptrade trying to figure out why the system dont work and send you a message of "Downgrading is not supported and is likely to cause unpredictable issues" and spend half an precious hour to find out is as simple as restart the memcached server to clear the cache

**Comment by solracsf** [comment]
Prefix of cached entries should change at version upgrades.
If this is not the case, please fill a complete bug report with error messages, if any.

By the way, Nextcloud can't simply restart your server, but only flush it if this was needed.

**Comment by joshtrichards** [comment]
Yeah, this sounds like a non-standard PHP OPCache configuration to me.

We document this fairly explicitly, including it's impact on upgrading and configuration changes. See [Server tuning - PHP OPCache - Revalidation](https://docs.nextcloud.com/server/latest/admin_manual/installation/server_tuning.html#revalidation). Particularly the *Warning* and *Danger* boxes.

I suppose we could add hint sort of like:

`Downgrading is not supported and is likely to cause unpredictable issues [...]. If this is unexpected, try restarting your PHP environment to clear your cache` or whatever

...at least to that particular exception.

**Comment by nextcloud-command** [comment]
This issue has been automatically marked as stale because it has not had recent activity and seems to be missing some essential information. It will be closed if no further activity occurs. Thank you for your contributions.
